lib/reducers/create-otp-reducer.js

```diff
@@ -17,7 +17,7 @@ const defaultQuery = {
   departArrive: 'NOW',
   date: getCurrentDate(),
   time: getCurrentTime(),
-  mode: 'TRANSIT,WALK'
+  mode: 'TRAM,BUS,GONDOLA,WALK'
```

Comment:
What happens when a mode supplied here doesn't exist in modes available in config?  It looks like `GONDOLA` cannot be removed from the list.  Perhaps the `modes-panel` should filter out any modes that are not available to prevent a mode from sneaking in and messing up plan request results.